hooks/useGetUserLocation.ts

```diff
@@ -0,0 +1,21 @@
+import { useState } from "react";
+
+export function useGetUserLocation() {
+  const [location, setLocation] = useState<
+    { latitude: number; longitude: number } | undefined
```

Comment:
Yukarida olusturdugum interface'i kullanabilirsin:

```suggestion
    ICoordinates | undefined
```